commons-text/src/main/java/org/jdbi/v3/commonstext/UseStringSubstitutorTemplateEngine.java

```diff
@@ -24,7 +24,7 @@
 @ConfiguringAnnotation(UseStringSubstitutorTemplateEngineImpl.class)
 @Target({ElementType.TYPE, ElementType.METHOD})
 public @interface UseStringSubstitutorTemplateEngine {
-    String prefix() default "${";
-    String suffix() default "}";
-    char escape() default '\\';
+    String prefix() default StringSubstitutorTemplateEngine.DEFAULT_PREFIX;
+    String suffix() default StringSubstitutorTemplateEngine.DEFAULT_SUFFIX;
+    char escape() default StringSubstitutorTemplateEngine.DEFAULT_ESCAPE;
```

Comment:
This refactor is fine, provided the Javadoc shows the literal values for those defaults.

Reply:
Is it ok like so?

Reply:
Yep.